tests/bootstrap.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once 'vendor/autoload.php';
+require_once(__DIR__ . '/../API/utils/ConnectionSettings.php');
+
+try {
+    $conn = new PDO(DB_DNS, $databaseUser, $databasePassword);
```

Comment:
This is currently using the same database settings as non-test environment, right? Can we rewrite the values before the tests run so that someone can run both test and non-test environments on the same machine? I would like to change the server, database, user, and password.